sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/ObjectAnchorsConversionClient.cs

```diff
@@ -0,0 +1,209 @@
+                ConversionConfiguration = options.ConversionConfiguration,
+                InputAssetUri = options.InputAssetUri
+            };
+
+            await _ingestionJobRestClient.CreateAsync(AccountId, options.JobId, body: properties, cancellationToken: cancellationToken).ConfigureAwait(false);
+            return new AssetConversionOperation(options.JobId, this);
+        }
+
+        /// <summary>
+        /// Retrieves an upload URI intended to house the model to be ingested
+        /// </summary>
+        /// <param name="cancellationToken">The cancellation token</param>
+        /// <returns><see cref="Response{GetAssetUploadUriResult}"/>.</returns>
+        public virtual Response<AssetUploadUriResult> GetAssetUploadUri(CancellationToken cancellationToken = default)
+        {
+            return _getBlobUploadEndpointRestClient.Get(AccountId, xMrcCv: GenerateCv(), cancellationToken: cancellationToken);
+        }
+
+        /// <summary>
+        /// Retrieves an upload URI intended to house the model to be ingested
+        /// </summary>
+        /// <param name="cancellationToken">The cancellation token</param>
+        /// <returns><see cref="Response{GetAssetUploadUriResult}"/>.</returns>
+        public virtual async Task<Response<AssetUploadUriResult>> GetAssetUploadUriAsync(CancellationToken cancellationToken = default)
+        {
+            return await _getBlobUploadEndpointRestClient.GetAsync(AccountId, xMrcCv: GenerateCv(), cancellationToken: cancellationToken).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Gets the status of an Object Anchors asset conversion job
+        /// </summary>
+        /// <param name="JobId">The asset conversion job's ID</param>
+        /// <param name="cancellationToken">The cancellation token</param>
+        /// <returns><see cref="Response{AssetConversionProperties}"/>.</returns>
+        internal virtual Response<AssetConversionProperties> GetAssetConversionStatus(Guid JobId, CancellationToken cancellationToken = default)
+        {
+            var accountId = AccountId;
+            var properties = _ingestionJobRestClient.Get(accountId, JobId, xMrcCv: GenerateCv(), cancellationToken: cancellationToken);
+            properties.Value.JobId = JobId;
+            properties.Value.AccountId = accountId;
+            return properties;
+        }
+
+        /// <summary>
+        /// Gets the status of an Object Anchors asset conversion job
+        /// </summary>
+        /// <param name="JobId">The asset conversion job's ID</param>
+        /// <param name="cancellationToken">The cancellation token</param>
+        /// <returns><see cref="Response{AssetConversionProperties}"/>.</returns>
+        internal virtual async Task<Response<AssetConversionProperties>> GetAssetConversionStatusAsync(Guid JobId, CancellationToken cancellationToken = default)
+        {
+            var accountId = AccountId;
```

Comment:
>var accountId = AccountId; [](start = 12, length = 26)

This assignment feels a bit unnecessary at this point.

Reply:
Removed

Reply:
Removed